lib/api/attributes.js

```diff
@@ -73,6 +73,75 @@ exports.attr = function(name, value) {
   return getAttr(this[0], name);
 };
 
+var getProp = function (el, name) {
+  return el.hasOwnProperty(name)
+      ? el[name]
+      : rboolean.test(name)
+          ? getAttr(el, name) !== undefined
+          : getAttr(el, name);
+};
+
+var setProp = function (el, name, value) {
+  el[name] = rboolean.test(name) ? !!value : value;
+};
+
+exports.prop = function (name, value) {
+  var i = 0,
+      property;
+
+  if (typeof name === 'string' && value === undefined) {
+
+    switch (name) {
+      case 'style':
```

Comment:
Cheerio's responsibilities stop at the jQuery API; mimicking DOM API's in this library is a slipperly slope. For instance, why `item()` but not `removeProperty()`? A more complete implementation might find it's home in [the DOMHandler library](https://github.com/fb55/domhandler) which would trickle down to Cheerio. It feels like an over-extension to insert a partial implementation here.